apps/src/code-studio/progressRedux.js

```diff
@@ -42,6 +42,8 @@ export default function reducer(state = initialState, action) {
       professionalLearningCourse: action.professionalLearningCourse,
       saveAnswersBeforeNavigation: action.saveAnswersBeforeNavigation,
       stages: action.stages.map(stage => _.omit(stage, 'hidden')),
+      // TODO - might make sense to split of some state that isnt actually specific to progress
+      scriptName: action.scriptName,
```

Comment:
I should remove this TODO, as I don't intend to address currently, but at some point it would be good to rationalize what info is stored in the progress subtree of our redux store.